scripts/us_census/pep/annual_population/clean.py

```diff
@@ -0,0 +1,542 @@
+            lambda row: FIPSCODE.get(row, "geoId/" + row))
+        if final_df is None:
+            final_df = next_df
+        else:
+            final_df = pd.concat([final_df, next_df], axis=0)
+    final_cols = ["Location"] + first_df_cols[3:8] + second_df_cols[3:8]
+
+    return final_df[final_cols]
+
+
+def _create_final_file(op_tmp_file: str, search_string: str,
+                       op_file: str) -> None:
+    """
+
+    Args:
+        op_tmp_file (str): Output temp File path
+        search_string (str): Search String in File
+        op_file (str): Output File
+    """
+    states = []
+    with open(op_tmp_file + ".csv", encoding='utf-8') as file:
+        for line in file.readlines():
+            if line.strip() == '':
+                continue
+            if line.strip().find(search_string) != -1:
+                header = ['Location']
+                header[1:] = (" ".join(line.strip('\n').split()).split())
+            elif len(" ".join(line.strip('\n').split()).split()[0]) == 2:
+                states.append(" ".join(line.strip('\n').split()).split())
+    with open(op_file + ".csv", 'a', encoding='utf-8') as op:
+        for state in states:
+            items = dict(zip(header, state))
+            # state_name = items['Location']
+            state_name = USSTATE_MAP.get(items['Location'], items['Location'])
+            for k, v in items.items():
+                if k not in ['Location', '(census)', '1970']:
+                    op.write("\n" + k + "," + state_name + "," +
+                             v.replace(",", ""))
+
+
+def _create_intermediate_file(ip_file: str, temp_file1: str, temp_file2: str,
+                              search_string1: str, search_string2: str) -> None:
+    """
+
+    Args:
+        ip_file (str): Input File Path
+        temp_file1 (str): Temporary File Path
+        temp_file2 (str): Temporary File Path
+        search_string1 (str): Search String in Input File
+        search_string2 (str): Search String in Input File
+    """
+
+    flag1 = 0
+    flag2 = 0
+
+    with open(temp_file1 + ".csv", 'w', encoding='utf-8') as temp_file_1:
+        with open(temp_file2 + ".csv", 'w', encoding='utf-8') as temp_file_2:
+            with open(ip_file, encoding='utf-8') as file:
+                for line in file.readlines():
+                    if line.strip() == search_string1:
+                        flag1 = 1
+                        flag2 = 0
+                    elif line.strip() == search_string2:
+                        flag1 = 0
+                        flag2 = 1
+
+                    if flag1:
+                        temp_file_1.write(line)
+                    elif flag2:
+                        temp_file_2.write(line)
+
+
+def get_state_file_config() -> dict:
+    """
+    This Method create a dict of states config file.
+    Returns:
+        states_config (dict): dict of states file config
+    """
+    states_config = {
```

Comment:
This should be a global constant or in a separate file with comments on what it means